tools/tar.wdl

```diff
@@ -0,0 +1,50 @@
+## Description: 
+##
+## This WDL tool wraps any functionality related to tar.
+
+version 1.0
+
+task tar_print_version {
+    command {
+        tar --version
+    }
+
+    runtime {
+        docker: 'stjudecloud/bioinformatics-base:bleeding-edge'
```

Comment:
Just a suggestion that we could probably just use ubuntu 18.04 images for these tasks which is what `bioinformatics-base` is based on. 

```suggestion
        docker: 'ubuntu:18.04'
```